src/NzbDrone.Core/Notifications/Webhook/WebhookPayload.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace NzbDrone.Core.Notifications.Webhook
+{
+    internal class WebhookPayload
+    {
+        public string EventType { get; internal set; }
+        public WebhookSeries Series { get; internal set; }
+        public WebhookEpisodeFile EpisodeFile { get; internal set; }
+        public WebhookEpisode Episode { get; internal set; }
```

Comment:
Scene name is usually the release (folder) name, but in a season pack it could be the file name. Its very unlikely that a season pack would have multiple qualities though.